### ISSUE TITLE
Podcast Ownership Frontend 

### ISSUE THREAD

**Issue by mkcode** [MLH-Fellowship/dev.to#85]
Continuing on #81, How should display podcast ownership on the site if at all?

For non-admin sections:

- Display the owner name for every podcast on the `podcast_episodes#index`page
- Anything else? Followings page?

For admin sections: Shall we wait on this while the admin section is in flux?

**Comment by mkcode** [comment]
@benhalpern - Can you provide input here?

**Comment by mkcode** [comment]
@cmgorton - Can you provide direction on this?

**Comment by cmgorton** [comment]
Yes! Looking in to it right now. Thanks for the ping.

**Comment by benhalpern** [comment]
Here are some outlines for overall frontend functionality, but with some general notes on closing the loop on this whole feature.

## For admins functionality:

We already have some of this implemented with the "role" of owner, and this overall functionality is designed to supplant that general-purpose use of the "role" with a more specialized join table.

I actually didn't realize that this part of things was basically already built out, so if you could replace the concept of "podcast admin", aka the "role", with the join table that we have with this new PR.

On the admin controller functionality and view, this should be enough. We already have the ability to add and remove this de facto rule. What we don't have is more functionality for owners to do.

## For podcast owners (who are not admins)

Podcast owners should be able to access an `/edit` path with podcast as well as podcast episode, where they have a form to edit any details public of that show or episode.

The edit views should use the crayons design system to create a form which is visually similar to what we have in `/settings` etc. But since this feature can be in beta, we can take whatever your best effort is for initial implementation and our designers can come back to it after.

Within the `/edit` view of the podcast itself (not the episode), owners should be able to add and remove other owners, similar to what we can do in admin. This is, of course, only available to _existing_ owners, so we don't need a path for the first owner to be created.

This feature should fully replace the existing "role" and have those users migrated using a "data update script"

https://docs.forem.com/backend/data-update-scripts/
